src/main/AndroidManifest.xml

```diff
@@ -39,9 +39,18 @@
       </intent-filter>
     </activity>
     <activity android:name="EmbeddedBrowserActivity"
-        android:screenOrientation="portrait"
-        android:configChanges="orientation|screenSize"
-        tools:ignore="DiscouragedApi"/>
+      android:screenOrientation="portrait"
+      android:configChanges="orientation|screenSize"
+      android:launchMode="singleTask"
```

Comment:
One subtle change here is switching the [launchMode](https://developer.android.com/guide/topics/manifest/activity-element#lmode) from `singleInstance` to `singleTask`.  As far as I could tell, we always had `singleInstace` set for the `AppUrlIntentActivity`, but just reading the documentation, it feels like that is not really the right choice.  With the way I currently understand things, `singleTask` will allow us to always re-use the existing instance of `EmbeddedBrowserActivity` _and continue the task chain as normal_.  The `singleInstance` docs note that any additional activities launched off of the root are not part of the same task chain.

TLDR, as far as I can tell, `singleTask` is "more normal" than `singleInstance` and we get the desired behavior.